doc/usecases/core/UC_key_value.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: `Key` Value
+
+## Summary
+
+- **Title:** Create `Key`
+- **Scope:** `libelektra-core`
+- **Level:** Developer Goal
+- **Actors:** Core, Caller
+- **Brief:** Access to and manipulation `Key`'s value
+
+## Scenarios
+
+- **Precondition:**
+  - [`Key` has been created](UC_key_create.md)
+- **Main success scenario:**
+  - Caller requests current value of `Key`
+  - Core returns current value of `Key`
+- **Alternative scenario:**
+  - Caller requests to change value of `Key` to new value
+  - Core stores new value in `Key`
+- **Error scenario:** -
```

Comment:
If not string/binary as requested.

Reply:
See above. This is actually another reason why I avoided the distinction. It creates unnecessary error cases. The type error is on a higher level than this use case IMO. If you want I can write a separate use case for typed value access. Which types are supported would be a technical implementation detail.